src/textual/css/tokenize.py

```diff
@@ -152,15 +161,40 @@ class DeclarationTokenizerState(TokenizerState):
     }
 
 
+class ValueTokenizerState(TokenizerState):
+    EXPECT = expect_declaration_content_solo
+
+
 tokenize = TokenizerState()
 tokenize_declarations = DeclarationTokenizerState()
+tokenize_value = ValueTokenizerState()
+
+
+def tokenize_values(values: dict[str, str]) -> dict[str, list[Token]]:
+    """Tokens the values in a dict of strings.
+
+    Args:
+        values (dict[str, str]): A mapping of name on to a CSS value.
```

Comment:
Could you please clarify what you mean by name here by updating this docstring? Is it CSS property name, or style object attribute name?

Reply:
It's a css variable name. Updated the docstring to say that.